scripts/winesapLiveInstaller.sh

```diff
@@ -0,0 +1,892 @@
+#!/bin/bash
```

Comment:
Thanks for creating this!

It appears to be an interactive alternative to `winesapos-install.sh`. It is missing all of the customizations of winesapOS and is more of a generic Arch Linux installer.

The reason our dual-boot approach uses a root file system of the minimal image is because those packages and configurations have been thoroughly tested and it is a 1:1 replication of what we provide. I'm already exploring the re-use of the installer used by ChimeraOS and PlaytronOS. It will `dd` our minimal image to an internal drive.

I suggest moving this script to a separate PR. Then we can discuss how to integrate this into `winesapos-install.sh` or elsewhere.